script/reproducible/verify.sh

```diff
@@ -0,0 +1,195 @@
+
+    which -s brew
+    if [[ $? != 0 ]] ; then
+        # Install Homebrew
+        ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+    else
+        brew update
+    fi
+
+    which -s python2
+    if [[ $? != 0 ]] ; then
+        # Install python2
+        brew install python2
+    fi
+
+    which -s python3
+    if [[ $? != 0 ]] ; then
+        # Install python3
+        brew install python3
+    fi
+
+    if [ ! -f /usr/local/bin/gtar ]; then
+      brew install gnu-tar
+      ln -fs /usr/local/bin/gtar /usr/local/bin/tar
+    fi
+
+    XCODE_VERSION=$(curl -v https://raw.githubusercontent.com/electron/electron/${CHECK_VERSION}/.circleci/config.yml | grep "xcode:"  | head -n1 | awk '{print $2}' | tr -d '"')
+    echo "Since there is no way to automate XCode installation. Install XCode version $XCODE_VERSION manually."
+    echo "Press Key to continue."
+    open 'https://developer.apple.com/download/more/?q=xcode'
+    read -n1 ans
+}
+
+nocheckdisk=false
+
+while getopts ":b:n" o; do
+    case "${o}" in
+        b)
+            buildir=${OPTARG}
+            ;;
+        n)
+            nodiskcheck=true
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+export CHECK_VERSION=$1
+
+if [[ "$OSTYPE" == "linux-gnu" ]] && cat /proc/1/cgroup | grep docker &>/dev/null; then
+  # If you are in docker run youself and that's it
+  echo "Executing under docker image, continuing"
+  git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git $HOME/depot_tools
+  export GCLIENT_PY3=0
+  export PATH=$HOME/depot_tools:$PATH
+elif [[ "$OSTYPE" == "linux-gnu" ]] && ! (cat /proc/1/cgroup | grep docker &>/dev/null); then
+  # If you are in linux run yourself but inside the correct container
+  IMAGE=$(curl -v https://raw.githubusercontent.com/electron/electron/${CHECK_VERSION}/.circleci/config.yml | grep "image:" | awk '{print $3}')
+  docker run -ti -v ${buildir}:${buildir} ${EXTRA_DOCKER_ARGS} -v ${SCRIPT}:/verify.sh ${IMAGE} bash -e /verify.sh -b ${buildir} $([ "$nodiskcheck" = "true" ] && printf " -n") ${CHECK_VERSION}
+  exit 0
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+  # If you are on mac run youself pointing to the correct version of toolchain
+  setup_mac_for_dev "${CHECK_VERSION}" || true
+  echo "Executing under macOS image, continuing"
+fi
+
+export ELECTRON_OUT_DIR=Default
+export ELECTRON_ENABLE_STACK_DUMPING=1
+
+echo Check for disk space
+export FREESPACE=$(BLOCKSIZE=1g df $buildir | tail -n1 | awk '{print $4}')
+export MINSPACE=100 # number of gygabites needed to download and build electron
+
+if [ $FREESPACE -ge $MINSPACE ] || [ "$nodiskcheck" = "true" ]; then
+  echo yes enough free space
+else
+  echo not enough free space - ${MINSPACE}GB 
+  exit 5
+fi
+
+export GN_CONFIG=release
+export STRIP_BINARIES=true 
+export GENERATE_SYMBOLS=true
+export CHECK_DIST_MANIFEST=1
+
+echo "Building $env:GN_CONFIG build"
```

Comment:
```suggestion
echo "Building $GN_CONFIG build"
```

Reply:
Also, the `GN_CONFIG` variable doesn't seem to be used anywhere. Perhaps remove it?